src/main/java/com/rapid7/jul/LogentriesHandler.java

```diff
@@ -235,8 +136,19 @@ String getStringProperty(String name, String defaultValue) {
         return val.trim();
     }
 
-    byte[] getBytesProperty(String name, String defaultValue) {
-        return getStringProperty(name, defaultValue).getBytes();
+    boolean getBooleanProperty(String name, boolean defaultValue) {
+        LogManager manager = LogManager.getLogManager();
+        String val = manager.getProperty(name);
+        if (val == null) {
+            return defaultValue;
+        }
+        try {
+            return Boolean.parseBoolean(val.trim());
+        } catch (NumberFormatException e) {
```

Comment:
This exception isn't thrown

Reply:
```
   public static boolean parseBoolean(String s) {
        return ((s != null) && s.equalsIgnoreCase("true"));
    }
```

Anything other than "true" (case-insensitive) will disable SSL. In this case, maybe it's better to do something like `"false".equalsIgnoreCase(val.trim())`